static/src/javascripts/bootstraps/common.js

```diff
@@ -496,11 +498,31 @@ define([
                         }).show(template(internationalControlMessage, {}));
                     }
                 }
+            },
+
+            initSaveForLater: function () {
+                // On top of the A/B test, we always want to give pre-existing SFL
+                // users the web feature.
+                mediator.on('module:identity:api:loaded', function () {
+                    id.getSavedArticles().then(function (resp) {
+                        var userHasSavedArticles = !! resp.savedArticles;
+
+                        var test = ab.getTest('SaveForLater');
+                        var participation = ab.isParticipating(test);
+                        var userIsNotInTest = !participation || !ab.getVariant(test, participation.variant);
+                        if (userIsNotInTest && userHasSavedArticles) {
```

Comment:
... and `!userIsInTest` ? Also if the user is in the test, and has saved articles, shouldn't the SFL module be loaded?